v6_carrier_py/wrapper.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pickle
+from io import StringIO
+
+import pandas as pd
+from SPARQLWrapper import SPARQLWrapper, CSV
+from vantage6.client import serialization
+from vantage6.tools import docker_wrapper
+from vantage6.tools.data_format import DataFormat
+from vantage6.tools.dispatch_rpc import dispact_rpc
+from vantage6.tools.util import info
+
+SPARQL_RETURN_FORMAT = CSV
+
+
+def sparql_wrapper(module: str):
```

Comment:
Can you add some documentation here? What is module?

Reply:
It would also be good to document what the query should look like.

Reply:
I specified that it should be a SPARQL `SELECT` query string. That seems enough to me.

Reply:
Nice, I really like the documentation! It would have helped me if you specified what the query should return a bit more, but I'll address that in a different comment.